content/en/metrics/guide/metrics-without-limits-getting-started.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Getting Started with Metrics without Limits™
+kind: guide
+is_beta: true
+---
+
+<div class="alert alert-warning">This functionality is in private beta.</div>
+
+## Overview
+
+This guide covers how to get started with Metrics without Limits™ and quickly gain the most value from the feature. 
+
+Review the [Metrics without Limits™ FAQ][1].
+
+
+1. Begin by configuring your Top 20 metrics on the Usage page.
+   You can [configure metrics][2] both in the Metrics Summary UI and with the API.
+
+   **Note:** If you're using the Create Tag Configuration API, use the [tag configuration cardinality estimator API][3] first to validate the potential impact of your tag configurations prior to using the API above to create tag configurations. 
```

Comment:
If they should do this first, then this instruction should come before the step on configuring. Maybe:

1. Review the MwL FAQ.
2. If you are going to use the API, use the estimator
3. Configure your top twenty metrics using the UI or the API.

Reply:
+

Reply:
Reordered 👍🏻